.github/workflows/ci-docs.yaml

```diff
@@ -46,7 +46,13 @@ jobs:
   e2e-tests:
     strategy:
       matrix:
-        os: [[self-hosted, macos, amd64, 13], [self-hosted, macos, amd64, 12.6], [self-hosted, macos, arm64, 13], [self-hosted, macos, arm64, 12.6]]
+        os: 
+          [
+            [self-hosted, macos, amd64, 13, test], 
+            [self-hosted, macos, amd64, 12, test], 
```

Comment:
What is the implication of the value of `test`/`release`? Is it possible to add one line of comment explaining what is the implication of these values?

Reply:
The implication is that workflows that are not part of the release process should be labelled 'release'. This way, we can separate our testing and release infrastructure to increase security. Do you think we should add a comment to every file where this occurs? Building on your idea a bit, it might be better instead to have a central markdown file to explain all the labels.

Reply:
Central markdown file is fine. As long as it is easy to find. 😄